### ISSUE TITLE
Refactoring of solChSpecF

### ISSUE THREAD

**Issue by hmadej** [JacquesCarette/literate-scientific-software#316]
Ok the following changes must be made to each example:
** denote changes
```
solChSpecF :: (NamedIdea a) => a -> (Section, Section) **-> Bool ->** Sentence -> (Sentence, Bool, Sentence) -> 
  ([Contents], [Contents], [Contents], [Contents], [Contents], [Contents]) -> [Section] -> Section

```
is now:

```
solChSpecF :: (NamedIdea a) => a -> **(Section, Section) -> Sentence ->** (Sentence, Bool, Sentence) -> 
  ([Contents], [Contents], [Contents], [Contents], [Contents], [Contents]) -> [Section] -> Section
```

All examples implementing solChSpecF must remove the Boolean from there function.

- [ ] SSP
- [x] GlassBR
- [x] SWHS
- [x] NoPCM
- [x] GamePhys

In addition GlassBR and SWHS must also change the following in there data constraint sections:

Remove sentence from list of data constraint tables
Add that sentence to the trailing sentence for data constraints.

You may have to remove Paragraph $ or change foldlSP to foldlSent 

- [x] GlassBR
- [x] SWHS

specifically for SWHS: 
```
s4_2_6_DataConTables :: [Contents]
s4_2_6_DataConTables = [s4_2_6_table1,  **s4_2_6_T1footer**, s4_2_6_table2, s4_2_6_table3]
```
s4_2_6_T1footer must be removed, changed to a Sentence from Contents, and replace:

```
        end = sParen (foldlSent [S "The", plural table_, S "are left out",
          S "because features they should use are not yet implemented",
          S "in Drasil"])
```

with it, as the tables are implemented.

for glassBR:
remove 
`[s6_2_5_table1, s6_2_5_table2, **s6_2_5_intro2,** s6_2_5_table3]`

change s6_2_5_intro2 so it uses foldlSent instead of foldlSP or Paragraph $ foldlSent
change s6_2_5_intro2 type declaration to Sentence, and add s6_2_5_intro2 to end:

```
  where end = foldlSent [(makeRef s6_2_5_table2), S "gives", 
             (plural value `ofThe` S "specification"), plural parameter, 
              S "used in" +:+. (makeRef s6_2_5_table1), getS ar_max, --FIXME: Issue #167
              S "refers to the", phrase ar_max, S "for the plate of glass"] **+:+ s6_2_5_intro2**
```

**Comment by danscime** [comment]
SWHS as of 0a0d806

**Comment by AKM11** [comment]
NoPCM fixed as of commit 53048be2e7e7c0f2e04bd4596c2c2763d7af1149.

**Comment by hmadej** [comment]
everyone has made change to body.
